scalableminds/webknossos
# Upgrade to Yarn 4

Updated yarn to version 4 following the [offical migration guide.](https://yarnpkg.com/migration/guide). Except for the `.gitignore`file all files were auto-created/auto-updated.

Other changes:
- Removed some additional `yarn` installation of the `command`-JS-package during the SBT asset compilation. The package is already installed as a dev-dependency of the main yarn WK installation.
- Remove yarn caching from CI. `Restore Cache` (51s) and `Save Cache` (2 min) are slower than just `yarn install` including the download (18s). 

### Steps to test:
1. Make sure that `node` 18+ is installed
2. Run `corepack enable`
3. Run `yarn install`

### Issues:
- fixes #8051

------
(Please delete unneeded items, merge only when none are left open)
